types/docusign-esign/index.d.ts

```diff
@@ -109,6 +109,7 @@ export class ApiClient {
 }
 
 export class Configuration {
+    static default: Configuration;
```

Comment:
there is a static property on the class that will construct an instance of the configuration class